pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -680,6 +682,14 @@ func cleanUpController(ctx context.Context, newSCSIController types.BaseVirtualD
 	return nil
 }
 
+// Get datastore from VMDisk path.
+// vmDiskPath format is [<datastore> kubevols/<volume>]
+func getDatastoreFromVMDiskPath(vmDiskPath string) string {
```

Comment:
We should update the vendor sw and pick up this change https://github.com/vmware/govmomi/commit/e152c3553e1101445babdb94082a36df9fd3ccf3